MicrosoftTeams.schema.json

```diff
@@ -185,6 +185,23 @@
               ]
             }
           },
+          "context": {
+            "type": "array",
+            "description": "The set of contextItem scopes that a tab belongs to",
+            "maxItems": 8,
+            "items": {
+              "enum": [
+                "personalTab",
+                "channelTab",
+                "privateChatTab",
```

Comment:
looks like these `Enums` are new:
* privateChatTab
* meetingChatTab
* meetingDetailsTab
* meetingSidePanel
* meetingStage
* callingSidePanel

Please make sure that we have support of them in MiddleTier. @Anoop-D fyi

Reply:
@greybax Thanks for checking, this property has already been added in MiddleTier
https://domoreexp.visualstudio.com/Teamspace/_git/Teamspace-MiddleTier/pullrequest/248031?_a=overview

Reply:
Yes these enum values are available in Middle Tier